_targets/build.project.armv7m7-imxrt106x

```diff
@@ -95,34 +94,36 @@ b_build_target() {
 
 	b_log "Relink plo to include the new script"
 	(cd plo && make $MAKEFLAGS all)
+
+	cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET}.img" _boot/
 }
 
 
 b_add2img() {
-	printf "Copying %s (offs=%dB)\n" $1 $OFFSET
-	dd if=$1 of=$2 bs=1 seek=$OFFSET conv=notrunc 2>/dev/null
-	SZ=$((($(stat -c "%s" $1) + $SIZE_PAGE - 1)&$PAGE_MASK))
-	OFFSET=$(($OFFSET+$SZ))
+	printf "Copying %s (offs=%dB)\n" "$1" "$OFFSET"
+	dd if="$1" of="$2" bs=1 seek="$OFFSET" conv=notrunc 2>/dev/null
+	SZ=$((($(stat -c "%s" "$1") + SIZE_PAGE - 1) & PAGE_MASK))
```

Comment:
You're doing the same thing at least 3 times (lines:57, 61, 105), it would be better to make it a function (like `file_size_rounded` or similar)